### PR TITLE
[bwip-js] Fix alttext type - should be string

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: types/bwip-js/index.d.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ declare namespace BwipJs {
         paddingheight?: number | undefined;
 
         monochrome?: boolean | undefined;
-        alttext?: boolean | undefined;
+        alttext?: string | undefined;
```
Comment (RyanNerd) [inline types/bwip-js/index.d.ts:37]: This looks like a bool from [this code](https://github.com/metafloor/bwip-js/blob/9fc4a7dba6b190eabd6461cf3866a97e179afc23/src/exports.js#L236) but it's not. It's a string that is when present sets `includetext = true`

This PR fixes a bug.

**Review by RyanNerd — APPROVED** [review]

File: types/bwip-js/index.d.ts (hunk 1 above)
Comment (mikehalmamoj) [reply types/bwip-js/index.d.ts:37]: 👍 

I have tested the fix too in my local env and I was able to change the text beneath a barcode. Got stuck trying to write a test in this project for it though 🤷

Comment (RyanNerd) [reply types/bwip-js/index.d.ts:37]: I got tests working for this once many moons ago. I'll see if I can find some time today to script something.

File: types/bwip-js/tslint.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
         "no-consecutive-blank-lines": false,
         "no-padding": false,
         "no-unnecessary-type-assertion": false,
+        "no-outside-dependencies": false,
```
Comment (andrewbranch) [inline types/bwip-js/tslint.json:8]: Accidental/unnecessary change?

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/bwip-js/tslint.json (hunk 2 above)
Comment (RyanNerd) [reply types/bwip-js/tslint.json:8]: I approved the `alttext?: string | undefined;` change, but recommend any changes to the linter config be done as a separate PR.

Comment (mikehalmamoj) [reply types/bwip-js/tslint.json]: Gah! Sorry my bad, I had to set this to run tests locally and didn't mean to push it. Fix coming.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## bwip-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/bwip-js@latest/))
was missing the following properties:
1. request
1. render
1. raw
1. fixupOptions
1. BWIPJS_VERSION

<details>
<summary>as well as these 6 other properties...</summary>
<p>BWIPP_VERSION, BWIPJS, STBTT, FontLib, DrawingBuiltin, DrawingZlibPng</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d2d1692196faabf068418950665e46b9460e1f1c
</p>

**Comment by RyanNerd** [comment]
Add the following code to the bottom of `bwip-js-tests.ts`

```typescript
// Test the alttext fix as a browser implementation
const canvasElement2 = document.createElement("canvasTextTest") as HTMLCanvasElement;
canvasElement2.setAttribute("id", "canvasTextTest");
const toCanvasText = bwipjs.toCanvas("canvasTextTest", {
    bcid: "code128",
    text: "1123581321345589",
    alttext: "Alternative text"
});
toCanvasText; // $ExpectType HTMLCanvasElement
```

I forked your repo modified this and ran `npm run test bwip-js` with no errors.

**Review by RyanNerd — APPROVED** [review]
All looks good to me. Thx for the PR!

**Comment by mikehalmamoj** [comment]
Ready to merge
